integration-cli/docker_cli_build_test.go

```diff
@@ -4623,7 +4623,12 @@ func (s *DockerCLIBuildSuite) TestBuildMultiStageArg(c *testing.T) {
 	result.Assert(c, icmd.Success)
 
 	result = cli.DockerCmd(c, "images", "-q", "-f", "label=multifromtest=1")
-	parentID := strings.TrimSpace(result.Stdout())
+	result.Assert(c, icmd.Success)
+
+	imgs := strings.Split(strings.TrimSpace(result.Stdout()), "\n")
+	assert.Assert(c, is.Len(imgs, 1), `only one image with "multifromtest" label is expected`)
```

Comment:
One small change here:
Changed `Check` to `Assert` to:

1. Avoid bounds check panic when this would return no images
2. Avoid having this test having success/failure randomly depending on how the image ID is sorted